src/schema/type/MusicPlaylist.graphql

```diff
@@ -213,4 +213,6 @@ type MusicPlaylist implements SearchableInterface & ThingInterface & ProvenanceE
   numTracks: Int #aggregate from track property
   "https://schema.org/track"
   track: MusicRecording @relation(name: "MUSIC_RECORDING", direction: "OUT")
+  "https://schema.org/track"
+  trackItemList: ItemList @relation(name: "ITEM_LIST", direction: "OUT")
```

Comment:
maybe we should make this relation name a bit more explicit? `TRACK_ITEM_LIST`?

Reply:
This is one of the cases where we might need special consideration in the json-ld renderer. In this case we should add one or the other, but not both, and the value of the `track` field would have a different format

Reply:
Yes good point, I blindly followed the convention from above